test/e2e_node/e2e_node_suite_test.go

```diff
@@ -179,7 +179,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// This helps with debugging test flakes since it is hard to tell when a test failure is due to image pulling.
 	if framework.TestContext.PrepullImages {
 		klog.Infof("Pre-pulling images so that they are cached for the tests.")
-		updateImageWhiteList()
+		updateImageAllowList()
```

Comment:
I think both of these are the wrong term https://github.com/kubernetes/kubernetes/issues/92317#issuecomment-646825052, `updateImagePrepullList`?

Is there anywhere we use this as a filter? Or only for pre-pulling.

Reply:
Been staring at /test for a while at it seems to be that it is used only for prepulling.

As per changing the name to `updateImagePrepullList` YES and thank you for the suggestion.
Naming is hard :sweat_smile: